mlflow/tracking/fluent.py

```diff
@@ -742,6 +795,45 @@ def search_runs(
         are expanded into their own columns named metrics.*, params.*, and tags.*
         respectively. For runs that don't have a particular metric, parameter, or tag, their
         value will be (NumPy) Nan, None, or None respectively.
+
+    .. code-block:: python
+        :caption: Example
+
+        import mlflow
+
+        # Create an experiment and log two runs under it
+        experiment_id = mlflow.create_experiment("Social NLP Experiments")
+        with mlflow.start_run(experiment_id=experiment_id):
+            mlflow.log_metric("m", 1.55)
+            mlflow.set_tag("s.release", "1.1.0-RC")
+        with mlflow.start_run(experiment_id=experiment_id):
+            mlflow.log_metric("m", 2.50)
+            mlflow.set_tag("s.release", "1.2.0-GA")
+
+        # Search all runs in experiment_id
+        df = mlflow.search_runs([experiment_id], order_by=["metrics.m DESC"])
+
+        # Print pandas DataFrame's rows and columns
+        print(df.loc[:, ["metrics.m", "tags.s.release", "run_id"]].to_string())
```

Comment:
```suggestion
        print(df[["metrics.m", "tags.s.release", "run_id"]].to_string())
```

to simplify the code.

Reply:
I suppose you can index into pandas DataFrame with a list of columns too. Works fine. Will change it.